### ISSUE TITLE
[BUG]: Textures are broken on the car selection screen in Axel Impact - The Extreme Racing (Korea) with hardware renderers 

### ISSUE THREAD

**Issue by drachenschanze** [PCSX2/pcsx2#6897]
### Describe the Bug

The textures on the car and woman are broken and flickering when using the vulkan renderer. Happens at native resolution. 

### Reproduction Steps

Start the game and get to the car selection screen.

### PCSX2 Revision

pcsx2-v1.7.3229-windows-64bit-AVX2-Qt

### Operating System

Windows 10 (64bit)

### CPU

Intel i5

### GPU

GTX 1070

### GS Settings

Default

### Emulation Settings

Default

### GS Window Screenshots

![Uploading Axel Impact - The Extreme Racing_SCPS-56016_20220823153255.png…]()

### Logs & Dumps

[Axel Impact - The Extreme Racing_SCPS-56016_20220823152856.zip](https://github.com/PCSX2/pcsx2/files/9399634/Axel.Impact.-.The.Extreme.Racing_SCPS-56016_20220823152856.zip)

**Comment by refractionpcsx2** [comment]
have you got gamefixes disabled? we do have a gamefix in this game (and DT Racer) in order to address this, maybe it's not enough in the Korean version, might need the EE timing hack or something as well, idk.

**Comment by drachenschanze** [comment]
@refractionpcsx2 Game fixes are on.

**Comment by refractionpcsx2** [comment]
ahh, the game needs mipmap basic (at least), not sure if that's okay ingame, but it's enough for that menu

Can you try Mipmapping Basic and see if that's enough for ingame also? if so we can change the GameDB to set that automatically in future versions.

**Comment by drachenschanze** [comment]
@refractionpcsx2 Mipmapping fixes the menu but the race is broken.

![Axel Impact - The Extreme Racing_SCPS-56016_20220824000419](https://user-images.githubusercontent.com/96800307/186179603-7ff1c233-9e28-4cbb-b7e0-d04794f9b079.png)

**Comment by refractionpcsx2** [comment]
yeah that looks like it's using depth for something, and doing a bad job of it.

**Comment by refractionpcsx2** [comment]
I had a play, you can completely fix this by using Frame Buffer Conversion hack, but it absolutely tanks the framerate.  Setting the CPU Sprite Render Size to 5 "kind of" fixes it but there is a bit of corruption on the screen when doing that.

Probably no perfect fix for this at the moment (without tanking the framerate)
